test/unit/lib/plugins/aws/package/compile/events/apiGateway/lib/validate.test.js

```diff
@@ -413,6 +413,20 @@ describe('#validate()', () => {
   });
 
   it('should process cors defaults', () => {
+    const expectedCors = {
```

Comment:
For all newly introduced changes we should implement tests based on `runServerless` utility